Notify/Notify/Notify/ViewModels/LocationSettingsPageViewModel.cs

```diff
@@ -172,6 +170,36 @@ private async void onBackButtonClicked()
 
         #endregion
 
+        #region Use_Current_Location
+
+        private bool m_IsUseCurrentLocation;
+        private string m_TmpLongitude;
+        private string m_TmpLatitude;
+        public bool IsUseCurrentLocation
+        {
+            get => m_IsUseCurrentLocation;
+            set
+            {
+                m_IsUseCurrentLocation = value;
+                OnPropertyChanged(nameof(IsUseCurrentLocation));
+
+                if (m_IsUseCurrentLocation)
+                {
+                    m_TmpLongitude = Longitude;
+                    m_TmpLatitude = Latitude;
```

Comment:
Use a different name than m_TmpLongitude / m_TmpLatitude.
This is the current longitude and latitude, so m_CurrentLongitude / m_CurrentLatitude is a better option here.

Reply:
It's not - I'm saving here the coordinates of the address that the user picked, so if he unchecks the 'use current location' checkbox I will still have the old coordinates.

Reply:
Then use another name.
Something like suggestedAddressLatitude

Reply:
m_SelectedAddressLongitude
m_SelectedAddressLatitude